services/issues/issues.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package issues
```

Comment:
Should use same as for other services (ex. `pull`)
```suggestion
package issue
```

Reply:
@lafriks  that could be another PR since it's not related with pull requests.